src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -487,7 +487,7 @@
     <value>The endpoint {endpointName} is missing the required 'Url' parameter.</value>
   </data>
   <data name="NoCertSpecifiedNoDevelopmentCertificateFound" xml:space="preserve">
-    <value>Unable to configure HTTPS endpoint. No server certificate was specified, and the default developer certificate could not be found.
+    <value>Unable to configure HTTPS endpoint. No server certificate was specified, and the default developer certificate could not be found or is out of date.
 To generate a developer certificate run 'dotnet dev-certs https'. To trust the certificate (Windows and macOS only) run 'dotnet dev-certs https --trust'.
```

Comment:
Am I correct to assume running 'dotnet dev-certs https --trust' replace the out-of-date dev cert? If so, this LGTM.

Reply:
Hmm @javiercn do you have more info on this? If not we can add another line to instruct the user to use `--clean` to remove any old certs

Reply:
+ @jkotalik 

Yes, I believe this is the case.